pyartcd/pyartcd/pipelines/release_from_fbc.py

```
@@ -0,0 +1,713 @@
+        # Create shipment configurations
+        timestamp = datetime.now(timezone.utc).strftime('%Y%m%d%H%M%S')
+        shipments_by_kind = {}
+
+        # Create image shipment (shared for all FBC builds)
+        if image_snapshot:
+            shipment_config = self.create_shipment_config('image', image_snapshot)
+            shipments_by_kind['image'] = shipment_config
+
+        # Create separate FBC shipment for each FBC build
+        fbc_counter = 1
+        for fbc_nvr, fbc_snapshot in fbc_snapshots.items():
```

Ruff (0.14.4)

Loop control variable `fbc_nvr` not used within loop body

(B007)